website/api/controllers/webhooks/receive-from-stripe.js

```diff
@@ -69,12 +69,14 @@ module.exports = {
     let subscriptionForThisEvent = await Subscription.findOne({stripeSubscriptionId: subscriptionIdToFind}).populate('user');
 
     let STRIPE_EVENTS_SENT_BEFORE_A_SUBSCRIPTION_RECORD_EXISTS = [
-      'invoice.created',
-      'invoice.finalized',
-      'invoice.paid',
-      'invoice.payment_succeeded',
-      'invoice.payment_failed',
-      'invoice.payment_action_required',
+      'invoice.created',// Sent when a user submits the billing form on /customers/new-license, before the user's biliing card is charged.
+      'invoice.finalized',// Sent when a user submits the billing form on /customers/new-license, before the user's biliing card is charged.
```

Comment:
Is it true?  Both of these fire and both requests come in?  I thought `finalized` happened later, maybe?  (Either I way I went ahead and approved)

Reply:
@mikermcneil The `invoice.finalized` event is sent immediately after an `invoice.created` event is sent.